root/utility/edit.js

```diff
@@ -119,7 +135,7 @@ export function editorMayApprove(edit: EditT, editor: ?EditorT): boolean {
     case EDIT_SERIES_EDIT:
       const oldOrderingType = get(edit, 'data.old.ordering_type_id', 0);
       const newOrderingType = get(edit, 'data.new.ordering_type_id', 0);
-      if (oldOrderingType != newOrderingType) {
+      if (oldOrderingType !== newOrderingType) {
```

Comment:
I used != there intentionally since I've seen some edit data store numbers as strings for no reason (though I don't remember if this particular edit type does).

Though if you want to do `String(oldOrderingType) !== String(newOrderingType)` then that's fine too, if more verbose.

Reply:
Oh, ok. Can revert :) (eventually we could fix those and the old data, but that sounds like an awful lot of effort for something small).